src/visual.ts

```diff
@@ -655,6 +656,26 @@ export class BulletChart implements IVisual {
         return null;
     }
 
+    // Implemented for old enums using space containing keys for example "Horizontal Left" which doesn't exist in current version
+    private static updateOrientation(settings: BulletChartSettingsModel, dataView: DataView): void {
```

Comment:
don't mutate args in a static function
give this function a return type or make it not static

Reply:
fixed in a089cf8